### PR TITLE
Fault in example

### PR DESCRIPTION
**Pull request by belgianrubs**
## Proposed change

There was a little fault in the example of the Media Player documentation.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @belgianrubs, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20723).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/media_player.markdown [hunk 1]
```diff
@@ -113,7 +113,8 @@ Documentation:
 Example of calling media_player service with title and image set:
 
 ```yaml
-entity_id: media_player.chromecast
+target:
+  entity_id: media_player.chromecast
 service: media_player.play_media
```
Comment (klaasnicolaas) [inline source/_integrations/media_player.markdown]: Maybe move this to above target?
```suggestion
service: media_player.play_media
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/media_player.markdown (hunk 1 above)
Comment (belgianrubs) [reply source/_integrations/media_player.markdown]: Yes makes more sense indeed IMO!

**Review by klaasnicolaas — APPROVED** [review]
✅ LGTM

**Pull request merged**